code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -563,7 +568,27 @@
 	..()
 
 
+/obj/item/jacobs_ladder
+	name = "jacob's ladder"
+	desc = "A celestial ladder that violates the laws of physics, but respects the coordinate plane."
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "ladder00"
+
+/obj/item/jacobs_ladder/attack_self(mob/user)
+	var/turf/T = get_turf(src)
+	var/ladder_x = T.x
+	var/ladder_y = T.y
+	to_chat(user, "You unfold the ladder. It extends much farther than you were expecting.")
+	for(var/i in 1 to world.maxz)
+		if(i == ZLEVEL_CENTCOM || i == ZLEVEL_TRANSIT)
+			continue
+		new /obj/structure/ladder/unbreakable/jacob(ladder_x, ladder_y, i)
+	qdel(src)
 
+/obj/structure/ladder/unbreakable/jacob
+	name = "jacob's ladder"
+	desc = "An indestructible celestial ladder that violates the laws of physics, but respects the coordinate plane."
```

Comment:
coordinate plane is not an IC friendly term

Reply:
You said that twice